linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -1691,18 +1729,48 @@ private[emitter] class FunctionEmitter(sjsGen: SJSGen) {
             redo(IdentityHashCode(newExpr))(env)
           }
 
-        case Transient(CallHelper(helper, args)) =>
-          unnest(args) { (newArgs, env) =>
-            redo(Transient(CallHelper(helper, newArgs)(rhs.tpe)))(env)
+        case Transient(SystemArrayCopy(src, srcPos, dest, destPos, length)) =>
```

Comment:
Given that `SystemArrayCopy` has type `NoType`, it can only appear in statement position. I believe it should be handled in `transformStat`, instead of here. It should also not be considered an expression in `isExpressionInternal`.

Reply:
That crashes the emitter :) Probably because the emitter assumes that the last element of a block is an expression. I'm not sure what is broken here: The optimizer that translates an `Apply` node to a statement or the emitter (maybe it should check the type of that last node in the block).